utils/utils.c

```diff
@@ -833,4 +863,87 @@ TEST_CASE(utils_check_datapath)
 
 	return TEST_OK;
 }
+
+#define TEST_PATH_NAME "for_testing_u81ijhgj0h1"
+
+static char *create_test_directory()
+{
+	int status;
+	char *path = NULL;
+	char current_working_dir[PATH_MAX];
+
+	getcwd(current_working_dir, PATH_MAX);
+	xasprintf(&path, "%s/%s", current_working_dir, TEST_PATH_NAME);
+
+	if (access(path, F_OK) != -1) {
+		rmdir(path);
+		status = mkdir(path, 0777);
+	}
+	else {
+		status = mkdir(path, 0777);
+	}
+
+
+	if (status < 0) {
+		pr_err("Error while trying to create dir '%s' [%d] \n%m\n",
+				path, status);
+	}
+
+	return path;
+}
+
+static char *create_and_write_info(char* dirpath, char *text)
+{
+	int fd;
+	char *path = NULL;
+
+	xasprintf(&path, "%s/info", dirpath);
+
+	if (access(path, F_OK) != -1) {
+		remove(path);
+		fd = open(path, O_RDWR|O_CREAT, 0777);
+	}
+	else {
+		fd = open(path, O_RDWR|O_CREAT, 0777);
+	}
+
+	if (fd < 0) {
+		pr_err("Error while trying to create file '%s' [%d] \n%m\n",
+				path, fd);
+	}
+	write(fd, text, sizeof(text));
+
+	return path;
+}
+
+TEST_CASE(utils_is_uftrace_dir)
```

Comment:
Well, I don't think it's needed testing with a fake directory.  Let' just get rid of the test case for this.